WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -123,7 +124,8 @@
           <Border Classes="warning">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction uses non-private coins." />
+              <TextBlock Text="Transaction uses non-private coins."
+                         ToolTip.Tip="Some entities will be able to deanonymize this transaction because when multiple coins are used for a transaction, the privacy of it is only as good as the lowest privacy of the coins used. You should conjoin first to have (enough) private coins for this transaction." />
```

Comment:
A shorter version of Turbolay's suggestion:

```suggestion
                         ToolTip.Tip="Some entities can deanonymize this transaction because its privacy is as good as the used coins' lowest privacy. Coinjoin more to have (enough) private coins for this transaction." />
```